JavaScript/JS-Week_1/Javascript_Homework/Housey Pricey/script.js

```diff
@@ -0,0 +1,31 @@
+// Peter's parameters
+let pWidth = 8;
+let pHeight = 10;
+let pDepth = 10;
+let pGardenSize = 100;
+let pRealPrice = 2500000;
+
+// Julia's parameters
+let jWidth = 5;
+let jHeigth = 8;
+let jDepth = 11;
+let jGardenSize = 70;
+let jRealPrice = 1000000;
```

Comment:
We worked with objects a bit during the session. Could a Julia object been used better here and would it have looked better? No need to update the code.

Reply:
Sorry for my mistake. It is true that it is not well understood. I have changed it.